### PR TITLE
=act #17898 Limit length of ByteString.toString

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Comment by patriknw** [comment]
Refs #17898

File: akka-actor/src/main/scala/akka/util/ByteString.scala [hunk 1]
```diff
@@ -409,6 +409,13 @@ sealed abstract class ByteString extends IndexedSeq[Byte] with IndexedSeqOptimiz
   override def indexWhere(p: Byte ⇒ Boolean): Int = iterator.indexWhere(p)
   override def indexOf[B >: Byte](elem: B): Int = iterator.indexOf(elem)
 
+  override def toString(): String = {
+    if (size > 100)
+      take(100).toString + "..."
```
Comment (ktoso) [inline akka-actor/src/main/scala/akka/util/ByteString.scala]: Would it be useful to `" [and $size more]..."`?

Comment (patriknw) [inline akka-actor/src/main/scala/akka/util/ByteString.scala]: sounds good

**Comment by patriknw** [comment]
@ktoso updated with "and more"

**Comment by ktoso** [comment]
LGTM

**Pull request merged**